src/xml-exporter/article-parser.ts

```diff
@@ -7,6 +7,7 @@ import {Article} from "../types/article";
 
 import { parseXML } from "./xml-utils";
 import {createRelatedArticleState} from "../model/related-article";
+import { createAffiliationsState } from "../model/affiliation";
 
 export function getArticleManuscript(article: Article): Manuscript {
```

Comment:
We 're missing coverage for this whole file so I've raised an issue to address this later on. https://github.com/libero/editor/issues/698